client/vaultclient/vaultclient.go

```diff
@@ -353,7 +352,7 @@ func (c *vaultClient) renew(req *vaultClientRenewalRequest) error {
 	var renewalErr error
 	leaseDuration := req.increment
 	if req.isToken {
-		// Set the token in the API client to the one that needs
+		// SetHeaders the token in the API client to the one that needs
```

Comment:
Was this a find-and-replace gone wild? There are some others like this in `nomad/vault.go` as well 😅

Reply:
Ah crap, I dunno how I didn't see this